addons/l10n_lt/data/account_tax_template_data.xml

```diff
@@ -0,0 +1,380 @@
+        <field name="tax_group_id" ref="tax_group_vat"/>
+        <field name="name">Reversed Sale VAT 21% (VAT25)</field>
+        <field name="description">21% VAT</field>
+        <field name="type_tax_use">sale</field>
+        <field name="amount" eval="21.0"/>
+        <field name="amount_type">group</field>
+        <field name="chart_template_id" ref="account_chart_template_lithuania"/>
+        <field name="children_tax_ids" eval="[(6, 0, [ref('account_tax_template_sales_reversed_positive_21'), ref('account_tax_template_sales_reversed_negative_21')])]"/>
+        <field name="sequence" eval="10"/>
+    </record>
+    <!-- Purchases -->
+    <!-- 0% VAT -->
+    <record id="account_tax_template_purchase_0_vat5" model="account.tax.template">
+        <field name="tax_group_id" ref="tax_group_vat"/>
+        <field name="name">Purchase 0% VAT (VAT5) Tax Exempt in the territory of Lithuania</field>
+        <field name="type_tax_use">purchase</field>
+        <field name="amount" eval="0.0"/>
+        <field name="amount_type">percent</field>
+        <field name="chart_template_id" ref="account_chart_template_lithuania"/>
+        <field name="account_id" ref="account_account_template_2441"/>
+        <field name="refund_account_id" ref="account_account_template_2441"/>
+        <field name="sequence" eval="10"/>
+    </record>
+    <record id="account_tax_template_purchase_0_vat14" model="account.tax.template">
+        <field name="tax_group_id" ref="tax_group_vat"/>
+        <field name="name">Purchase 0% VAT (VAT14) export of goods, transportation services</field>
+        <field name="description">0% VAT</field>
+        <field name="type_tax_use">purchase</field>
+        <field name="amount" eval="0.0"/>
+        <field name="amount_type">percent</field>
+        <field name="chart_template_id" ref="account_chart_template_lithuania"/>
+        <field name="account_id" ref="account_account_template_2441"/>
+        <field name="refund_account_id" ref="account_account_template_2441"/>
+        <field name="sequence" eval="10"/>
+    </record>
+    <record id="account_tax_template_purchase_0" model="account.tax.template">
+        <field name="tax_group_id" ref="tax_group_vat"/>
+        <field name="name">Purchase 0% VAT (VAT15)</field>
+        <field name="description">0% VAT</field>
+        <field name="type_tax_use">purchase</field>
+        <field name="amount" eval="0.0"/>
+        <field name="amount_type">percent</field>
+        <field name="chart_template_id" ref="account_chart_template_lithuania"/>
+        <field name="account_id" ref="account_account_template_2441"/>
+        <field name="refund_account_id" ref="account_account_template_2441"/>
+        <field name="sequence" eval="10"/>
+    </record>
+    <record id="account_tax_template_purchase_0_vat42" model="account.tax.template">
+        <field name="tax_group_id" ref="tax_group_vat"/>
+        <field name="name">Purchase 0% VAT (VAT42) (-)</field>
```

Comment:
just confirming, `(-)` isn't a typo here, right ?

Reply:
Actually its a typo:). Good catch. Only negative amounts should have `(-)`. Will update.